owncloud/owncloud.py

```diff
@@ -693,6 +817,100 @@ def share_file_with_user(self, path, user, **kwargs):
             )
         raise ResponseError(res)
 
+    def create_group(self, group_name):
+        """Create a new group via provisioning API.
+	   If you get back an error 999, then the provisioning API is not enabled.
```

Comment:
Let's introduce a new Error called ApiNotSupportedError or something similar.
The code would then throw that error in case 999 was returned.